src/backend/functions/projects.ts

```diff
@@ -64,11 +78,11 @@ const uniqueRelationArgs = Prisma.validator<Prisma.WBS_ElementArgs>()({
 });
 
 const convertStatus = (status: WBS_Element_Status): WbsElementStatus =>
-  ({
```

Comment:
I'm pretty sure the spacing here was previously done automatically by prettier using the settings specified in the package.json, I believe there was also a minor comma thing I noticed that prettier would also remove. This is no issue at all with the code, I just want to point it out.